src/block/Bell.php

```diff
@@ -158,4 +160,16 @@ public function ring(int $faceHit) : void{
 			$world->broadcastPacketToViewers($this->position, $tile->createFakeUpdatePacket($faceHit));
 		}
 	}
+
+	private function isValidFaceToRing(int $faceHit) : bool{
+		return (
+			$this->attachmentType->equals(BellAttachmentType::CEILING()) ||
+			($this->attachmentType->equals(BellAttachmentType::FLOOR()) && Facing::axis($faceHit) === Facing::axis($this->facing)) ||
+			(
+				$this->attachmentType->equals(BellAttachmentType::ONE_WALL()) ||
+				$this->attachmentType->equals(BellAttachmentType::TWO_WALLS())) &&
+				($faceHit === Facing::rotateY($this->facing, false) || $faceHit === Facing::rotateY($this->facing, true)
+			)
```

Comment:
The braces in this block are a mess. I can't make head or tail of what's going on.

Reply:
It's exactly the code you wrote

Reply:
My code:
![image](https://user-images.githubusercontent.com/14214667/212917830-1ba51f28-22c3-4930-88e0-518d43e1415a.png)
Your code:
![image](https://user-images.githubusercontent.com/14214667/212917868-c826a499-d344-45bc-b4fd-31043d4f93d7.png)

I don't know what happened to the formatting here, but it's a confusing mess.

Reply:
Your code has been successfully copied and pasted